proxy/proxy/spacewalk-proxy.spec

```diff
@@ -195,7 +195,7 @@ Summary:        Custom Channel Package Manager for the Spacewalk Proxy Server
 Group:          Applications/Internet
 Requires:       %{pythonX}
 Requires:       %{pythonX}-rhnlib >= 2.5.56
-Requires:       mgr-push >= 4.0.0
+Requires:       mgr-push >= 4.1.0
```

Comment:
Why? Is there a reason why we require 4.1 version?

Reply:
Same as above.

Reply:
Rolled back.